packages/web-components/@groupby/elements-products/src/products-base.ts

```diff
@@ -27,7 +27,7 @@ export default class ProductsBase extends Base {
   /**
    * The product data to be rendered.
    */
-  @property({ type: Array }) products: Product[] = [];
+  @dataInitializer() @property({ type: Array })products: Product[] = [];
```

Comment:
Same as the previous instance.

```suggestion
  @dataInitializer() @property({ type: Array }) products: Product[] = [];
```